app/lib/frontend/backend.dart

```diff
@@ -191,7 +198,7 @@ class GCloudPackageRepository extends PackageRepository {
         onPause: () => subscription.pause(),
         onResume: () => subscription.resume(),
         onCancel: () => subscription.cancel());
-    return controller.stream;
+    return controller.stream.cast<PackageVersion>();
```

Comment:
Instead, you could make `controller` a `StreamController<PackageVersion>`:
```dart
  final controller = new StreamController<PackageVersion>(...);
```
Then you wouldn't need to cast here.